spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/src/main/resources/application.properties

```diff
@@ -1,2 +1,3 @@
+#spring.cloud.gcp.spanner.projectId=<your-project-id>
```

Comment:
`project-id` notation here and in other properties.

Reply:
Sorry for nagging, but it would be great to swap out the `projectId` and `instanceId` by `project-id` and `instance-id`.

After that, LGTM

Reply:
Ping :)